src/gui/mainwindow.cpp

```diff
@@ -150,10 +150,10 @@ MainWindow::MainWindow(QWidget *parent)
 
     m_ui->actionOpen->setIcon(GuiIconProvider::instance()->getIcon("list-add"));
     m_ui->actionDownloadFromURL->setIcon(GuiIconProvider::instance()->getIcon("insert-link"));
-    m_ui->actionSetUploadLimit->setIcon(QIcon(":/icons/skin/uploadLimit.png"));
-    m_ui->actionSetDownloadLimit->setIcon(QIcon(":/icons/skin/downloadLimit.png"));
-    m_ui->actionSetGlobalUploadLimit->setIcon(QIcon(":/icons/skin/uploadLimit.png"));
-    m_ui->actionSetGlobalDownloadLimit->setIcon(QIcon(":/icons/skin/downloadLimit.png"));
+    m_ui->actionSetUploadLimit->setIcon(GuiIconProvider::instance()->getIcon("kt-set-max-upload-speed"));
+    m_ui->actionSetDownloadLimit->setIcon(GuiIconProvider::instance()->getIcon("kt-set-max-download-speed"));
+    m_ui->actionSetGlobalUploadLimit->setIcon(GuiIconProvider::instance()->getIcon("kt-set-max-upload-speed"));
+    m_ui->actionSetGlobalDownloadLimit->setIcon(GuiIconProvider::instance()->getIcon("kt-set-max-download-speed"));
```

Comment:
Yes, right now they are Plasma specific (and "kt-" means KTorrent). I thought at first about maps of our icon names to popular themes, but at the end decided that actual file names are not that important for us.

For the proper solution (especially after discussing in #6249), I believe, we have to un-bundle our icons, install them into the hicolor theme and that will alow theme designer to ship qBt icons with themes. But let us do this after switching to SVG icons. In the meantime, any KDE/Plasma use may enjoy cleaner context menu.